src/libopensc/card-cardos.c

```diff
@@ -1279,7 +1519,8 @@ cardos_logout(sc_card_t *card)
 		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_2C
 		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_3
 		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_4
-			|| card->type == SC_CARD_TYPE_CARDOS_V5_0) {
+			|| card->type == SC_CARD_TYPE_CARDOS_V5_0
+			|| card->type == SC_CARD_TYPE_CARDOS_V5_3) {
```

Comment:
clang-format suggested the following change:


```
-	if (card->type == SC_CARD_TYPE_CARDOS_M4_01
-		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_2
-		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_2B
-		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_2C
-		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_3
-		   	|| card->type == SC_CARD_TYPE_CARDOS_M4_4
-			|| card->type == SC_CARD_TYPE_CARDOS_V5_0
-			|| card->type == SC_CARD_TYPE_CARDOS_V5_3) {
+	if (card->type == SC_CARD_TYPE_CARDOS_M4_01 || card->type == SC_CARD_TYPE_CARDOS_M4_2 || card->type == SC_CARD_TYPE_CARDOS_M4_2B || card->type == SC_CARD_TYPE_CARDOS_M4_2C || card->type == SC_CARD_TYPE_CARDOS_M4_3 || card->type == SC_CARD_TYPE_CARDOS_M4_4 || card->type == SC_CARD_TYPE_CARDOS_V5_0 || card->type == SC_CARD_TYPE_CARDOS_V5_3) {
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->